PWGHF/Macros/computeFonllPlusPythiaPredictions.C

```diff
@@ -109,7 +109,7 @@ std::vector<std::string> splitString(const std::string& str, char delimiter)
 }
 
 //__________________________________________________________________________________________________
-std::array<TH1D*, 3> readFonll(std::string inFile, std::string histName)
+std::array<TH1D*, 3> readFonll(const std::string& inFile, const std::string& histName)
```

Comment:
while I understand that it's a good practice, this is a macro, and this function is superfast anyway since it's executed 3 times before the simulation itself, that is the slow part :-)